semantic-diff/semantic-diff.cabal

```diff
@@ -0,0 +1,72 @@
+cabal-version:       2.4
+
+name:                semantic-diff
+version:             0.0.0.0
+synopsis:            TODO
+description:         TODO
+homepage:            https://github.com/github/semantic/tree/master/semantic-diff#readme
+bug-reports:         https://github.com/github/semantic/issues
+license:             MIT
+license-file:        LICENSE
+author:              The Semantic authors
+maintainer:          opensource+semantic@github.com
+copyright:           (c) 2019 GitHub, Inc.
+category:            Language
+build-type:          Simple
+stability:           alpha
+extra-source-files:  README.md
+
+tested-with:
+  GHC == 8.6.5
+
+common common
+  default-language: Haskell2010
+  ghc-options:
+    -Weverything
+    -Wno-missing-local-signatures
+    -Wno-missing-import-lists
+    -Wno-implicit-prelude
+    -Wno-safe
+    -Wno-unsafe
+    -Wno-name-shadowing
+    -Wno-monomorphism-restriction
+    -Wno-missed-specialisations
+    -Wno-all-missed-specialisations
+    -Wno-star-is-type
+  if (impl(ghc >= 8.8))
+    ghc-options: -Wno-missing-deriving-strategies
+
+library
+  import: common
+  hs-source-dirs: src
+  exposed-modules:   Data.Diff
+                   , Data.Algebra
+                   , Data.Edit
+                   , Data.Functor.Classes.Generic
+                   , Data.JSON.Fields
+                   , Data.Term
+                   , Diffing.Algorithm
+                   , Diffing.Algorithm.RWS
+                   , Diffing.Algorithm.RWS.FeatureVector
+                   , Diffing.Algorithm.SES
+                   , Diffing.Interpreter
+                   , Prologue
```

Comment:
1. The `exposed-modules` field accepts formatting without commas:

    ```cabal
    exposed-modules:
      Data.Diff
      Data.Algebra
      …
    ```

    I find this style supports nicer diffing; could you apply it here please? 🙏

2. Could you please alphabetize the list of modules? It makes finding things and updating the list simpler, tho it is unfortunately rather manual at the moment (absent something like `cabal-fmt`).

3. Instead of exporting `Prologue`, could you please either a) remove it outright from this package, or b) move it to an `other-modules` field?